### ISSUE TITLE
DataLoader

### ISSUE THREAD

**Issue by nidhal1231** [HarryVolek/PyTorch_Speaker_Verification#36]
hello
I can't execute this part of code on Jupyter Notebook (there are no errors but I can't execute it an i works on google colab )
train_dataset = SpeakerDatasetEMRAIreprocessed()
train_loader = DataLoader(train_dataset, batch_size=N, shuffle=True, num_workers=num_workers, drop_last=True) 
Thank you for help
